server/routes/index.ts

```diff
@@ -42,15 +42,15 @@ import { API_PREFIX } from '../../common';
 
 export function defineRoutes(router: IRouter, esClient: IClusterClient) {
   const internalUserSchema = schema.object({
-    description: schema.string(),
+    description: schema.maybe(schema.string()),
```

Comment:
I didn't get the idea for optional `description`. There is no text input for description on the front end page. Is this field defined in the schema of ES index?

Reply:
hmm, somehow, the legacy API supports this description field, and is saved in ES, so I would prefer to keep this field.